src/apps/authorship/admin/options.py

```diff
@@ -0,0 +1,44 @@
+from anion.contrib import admin
+
+from ..models import ArticleContribution, Author, BookContribution, PatentContribution
+
+
+@admin.register(Author)
+class AuthorAdmin(admin.ModelAdmin):
+    """
+    Admin options of the `authorship.Author` model.
+    """
+
+
+class ContributionAdmin(admin.ModelAdmin):
+    """
+    Admin options common to all the models based on `authorship.Contribution`.
```

Comment:
Append "class" at the end of the docstring.

```suggestion
    Admin options common to all the models based on `authorship.Contribution` class.
```